### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by frompotenza**
## Related Issue(s)
  - Issue #1
  - Issue #2

## Other Dependent PRs
  - PR #2
  - PR #3

## Changes Made
- Describe the changes made in this PR.

## Rationale/Other Solutions Considered
- Explain the rationale behind this PR.
- Mention any alternative solutions considered (if any).

## Tests Done
- Describe the tests that have been performed.
- Mention any relevant test cases or scenarios.

## Optional Screen Recording
Attach a screen recording of test run(s) (if applicable).

## Required Wiki Changes (if any)
- List any changes that need to be made to the project's wiki (if applicable).

## Required install_dependencies.sh Changes (if any)
- List any new dependencies that need to be installed by the install_dependencies.sh script

## Related Branches (other than one that will be merged)
- List any related branches (if applicable).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -31,80 +31,3 @@ Ubuntu 20 this is ROS Noetic.
...
-  
-### Flashing Firmware
-
-If you have an Arduino you can wire it up to emmulate the thrusters on the AUV. Upload the 
-binary file by connecting the Arduino via USB (currently hardcoded assumption USB shows up as /dev/ttyACM0) and running 
-the generated make target:
-
-    catkin build --no-deps  propulsion --make-args propulsion_embedded_thrusters-upload
-    catkin build --no-deps  depth_sensor --make-args depth_sensor_embedded_depth_sensor-upload
-    catkin build --no-deps  imu --make-args imu_embedded_imu-upload
```
Comment (MrMondrian) [inline README.md:89]: can you add the flashing commands to the wiki

Comment (antoinedang) [reply README.md:89]: could you also link to the Wiki to read about installation/usage

**Review by antoinedang — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (frompotenza) [reply README.md:89]: does anyone actually use this? I thought it was just old stuff

**Comment by frompotenza** [comment]
Can I mark it as resolved @antoinedang @MrMondrian ?

**Comment by antoinedang** [comment]
for some reason my comment didnt post thats my bad: I said the README should refer directly to the Wiki instead of having the paragraph about getting started. Once thats good we can bypass

**Comment by frompotenza** [comment]
> for some reason my comment didnt post thats my bad: I said the README should refer directly to the Wiki instead of having the paragraph about getting started. Once thats good we can bypass

better?

**Pull request merged**